package/MDAnalysis/analysis/gnm.py

```diff
@@ -154,6 +158,45 @@ def order_list(w):
     return list_map
 
 
+def neighbour_generator(positions, cutoff):
```

Comment:
yeah, the contact matrix stuff here would be a good candidate for moving to something more general.  this is really common for so much analysis